cdp/cdp_parser.py

```diff
@@ -107,17 +107,17 @@ def get_orig_parameters(self, check_values=False, argparse_vals_only=True):
         """
         self._parse_arguments()
         
-        if not self.__args_namespace.parameters:
+        if not self.__args_namespace:
             return None
 
         parameter = self.__parameter_cls()
 
         # Remove all of the variables.
         parameter.__dict__.clear()
 
-        # if self.__args_namespace.parameters is not None:
+        # if self.__args_namespace is not None:
         parameter.load_parameter_from_py(
-            self.__args_namespace.parameters)
```

Comment:
Looking at [the code for `CDPParameter.load_parameter_from_py`](https://github.com/CDAT/cdp/blob/5d108bb3160b0667ac2bb8c1f021e3865e6302f8/cdp/cdp_parameter.py#L38), a path to the file passed in via `-p` (see my comment above) must be used. So just passing in a [`Namespace`](https://docs.python.org/3/library/argparse.html#argparse.Namespace) object will not work.